virtusize/src/main/java/com/virtusize/android/compose/ui/VirtusizeComposeViewModel.kt

```diff
@@ -64,8 +64,8 @@ internal class VirtusizeComposeViewModel : ViewModel() {
         product: VirtusizeProduct,
         virtusizeView: VirtusizeView,
     ) {
-        virtusize.setupVirtusizeView(product = product, virtusizeView = virtusizeView)
         virtusize.load(product)
+        virtusize.setupVirtusizeView(product = product, virtusizeView = virtusizeView)
```

Comment:
The orders shouldn't matter, but we can load first, and then bind it with the view later.

Reply:
OK